ads/index.js

```diff
@@ -73,6 +74,9 @@ function onAdsComplete (event) {
 			utils.log.info('Ad loaded in slot', event);
 			if (slotsRendered === 0) {
 				perfMark('firstAdLoaded');
+				if (!/spoor-id=0/.test(document.cookie)) {
```

Comment:
Should we use a different number here? I have a feeling the poor people with spoor-ids starting with 0 are getting a raw deal with all our sampling.